lib/administrate/field/url.rb

```diff
@@ -11,6 +11,10 @@ def truncate
         data.to_s[0...truncation_length]
       end
 
+      def html_options
+        @options[:html_options] || {}
+      end
```

Comment:
I have two questions.

1. Is the method name OK?
2. Should we automatically add `rel="noopener"` for security?

Reply:
1. I think the method name is okay — it's the same as what Rails calls it.
2. No, I think that'd be surprising for the end user.